apkid/rules/dex/obfuscators.yara

```diff
@@ -525,17 +524,26 @@ rule blackobfuscator : obfuscator
       (14 0? ?? ?? ?? ?? | B7 ??)    // const v(\d), 0x(\d+) or xor-int/2addr v(\d), v(\d)
       (B7 ?? | D7 ?? ?? ??)          // xor-int/2addr v(\d), v(\d) or xor-int/lit16 v(\d), v(\d), 0x(\d+)
     }
-    $movnop = {
```

Comment:
To verify things I checked it out on normal dex files without obfuscation and i saw that it not unique and present on there also:
![image](https://github.com/user-attachments/assets/e139c435-c259-4677-a8f9-a155414594a1)


That is why removed to avoid false positives